### PR TITLE
Display CHWL logo on Landing Page

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: https://www.pivotaltracker.com/story/show/178898883
### UserStory:
```
As a charity
To have a recognized brand
we want our logo to be displayed on landing page
```
### Changes Proposed:
- adds logo and mission statement to landing page
- adds Nexa font
- build layout according to wireframe
### Image
image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+
+@font-face{
+  font-family: "Nexa";
+  src: url("../src/assets/fonts/NexaLight.otf") format("opentype");
+  font-weight: normal;
```
Comment (tochman) [inline src/index.css:5]: I would like you to research how fonts can be changed using theming: https://v3.material-ui.com/customization/themes/#typography

File: src/views/IndexView.jsx [hunk 2]
```diff
@@ -1,11 +1,45 @@
...
+              data-cy="logo"
+              style={styles.image}
+              alt="Community Health West London"
+            />
+            <h3 data-cy="mission-statement" style={styles.statement}>
+              Our aim is to improve people's health and well-being
+            </h3>
+          </Box>
+        </Grid>        
+      </Grid>
```
Comment (tochman) [inline src/views/IndexView.jsx:22]: You can remove the Fragment wrapping `Grid`

Comment (SevaDer14) [reply src/views/IndexView.jsx:22]: done

File: src/index.css (hunk 1 above)
Comment (SevaDer14) [reply src/index.css:5]: Noted, we added chore in PT for that.

File: package.json [hunk 3]
```diff
@@ -44,5 +44,10 @@
   "devDependencies": {
     "cypress": "^7.6.0",
     "start-server-and-test": "^1.12.5"
+  },
+  "prettier": {
+    "tabWidth": 2,
+    "semi": false,
```
[the remark this answers is not in the document]
Comment (SevaDer14) [reply package.json:50]: Mistake. Semicolon reinstated

File: cypress/integration/visitorCanNavigateBetweenSPAViews.feature.js [hunk 4]
```diff
@@ -1,36 +1,38 @@
...
+    })
     it("is expected to display view subtitle", () => {
-      cy.get("[data-cy=header-subtitle]").should("contain.text", "Search");
-    });
-  });
-});
+      cy.get("[data-cy=header-subtitle]").should("contain.text", "Search")
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (SevaDer14) [reply cypress/integration/visitorCanNavigateBetweenSPAViews.feature.js:38]: Semicolon reinstated

File: cypress.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
-
+  "viewportWidth": 1920,
+  "viewportHeight": 1080,
   "baseUrl": "http://localhost:3001",
```
Comment (aonomike) [inline cypress.json:4]: We probably need to implement .env file so we can just use an environmental variable as a base URL, thoughts?

Comment (lukeperrin10) [reply cypress.json:4]: This is only for test environment for cypress. We only want to use the localhost port for this. When we setup the calls to the API we will set up an env file for these.

Comment (aonomike) [reply cypress.json:4]: Aaaah right, did not see the file is cypress.json. Thanks

**Review by tochman — APPROVED** [review]

**Pull request merged**